src/lib/components/creditCardBrandImage.svelte

```diff
@@ -1,12 +1,14 @@
 <script lang="ts">
+    import { isValueOfStringEnum } from '$lib/helpers/types';
     import { sdk } from '$lib/stores/sdk';
+    import { CreditCard } from '@appwrite.io/console';
 
     export let brand: string;
     export let width = 23;
     export let height = 16;
 
     function getCreditCardImage(brand: string, width = 46, height = 32) {
-        if (!brand) return '';
+        if (!isValueOfStringEnum(CreditCard, brand)) return '';
```

Comment:
Is it possible to change the type of the method parameter instead so the method can't be called with an invalid value?

Reply:
I definitely think that's the ideal approach. I was hesitant because there are a lot of places where the `CreditCardBrandImage` component is used and the change would be a little big. How about we tackle it in a separate PR?